### ISSUE TITLE
merge into sensu-plugins-sensu ?

### ISSUE THREAD

**Issue by luisdavim** [m4ce/sensu-plugins-stale-results#1]
Hi,

I think this check could be merged into https://github.com/sensu-plugins/sensu-plugins-sensu what do you think? It implements a feature that I think it's missing there and it makes sense to have these checks that use the Sensu API in one place.

What do you think?
@cwjohnston and @majormoses what do you guys think?

Luis

**Comment by m4ce** [comment]
I can prepare a PR for this. Let me know.

**Comment by majormoses** [comment]
@m4ce let me know I can help you push it through.

**Comment by m4ce** [comment]
Alright, I've opened the PR. Once that is merged, I will update the README in my repos to redirect users to the sensu-plugins-sensu repo. I will also delete my public gems.
